Dockerfile

```diff
@@ -28,4 +27,4 @@ ADD docker-startup.sh /app/docker-startup.sh
 ADD target/*.yaml /app/
 ADD target/pay-*-allinone.jar /app/
 
```

Comment:
This appears to be missing the line to add `ENTRYPOINT ["tini", "-e", "143", "--"]`

Reply:
Yes, yes it is. Good spot.